src/language-server/simple-ui.langium

```diff
@@ -92,6 +95,9 @@ Linebreak:
 CSSProperties:
     property=('text-color'|'font-size'|'width'|'height'|'background-color')':' value=Expression;
 
+CSSClasses:
```

Comment:
Suggestion:
Seems like CSSClasses is the only rule that uses [].

We could simplify the grammar by removing the 'classes' keyword.

Reply:
Let's talk about that on Monday

Reply:
Let's talk about that and other grammar stuff on Monday.